.github/workflows/release-edc.yml

```diff
@@ -0,0 +1,75 @@
+name: Create EDC Release
+on:
+  workflow_dispatch:
+    inputs:
+      edc_version:
+        description: 'Version string that is used for publishing (e.g. "1.0.0", NOT "v1.0.0"). Appending -SNAPSHOT will create a snapshot release.'
+        required: true
+        type: string
+
+
+env:
+  EDC_VERSION: ${{ github.event.inputs.edc_version || inputs.edc_version }}
```

Comment:
channeling my inner monk: maybe we want to rename the input `...inputs.component_version`, and the file to `release-component.yml` to make it generic